src/_includes/banner.html

```diff
@@ -1,5 +1,9 @@
 <div class="banner">
   <p class="banner__text">
-    Dart and Flutter return to Google I/O on May 10! <a href="https://io.google/2023/?utm_source=dart&utm_medium=embedded_marketing&utm_campaign=hpp_banner&utm_content=" class="no-automatic-external">Register now</a>
+    Dart 3 is out today, along with Google I/O!
+    <br>
+    <a href="https://medium.com/p/53f065a10635" class="no-automatic-external">Check out the blog post</a> and 
```

Comment:
Will this link still work once published?

Reply:
Yeah, the ID is what matters, e.g. for an already-published blog https://medium.com/p/547ef3d6459b takes you to the full `/dartlang/<title>-<id>` URL

Reply:
Cool, just wanted to make sure :)

Reply:
I just learned that today!